R/generate_simulated_data.R

```diff
@@ -187,7 +187,7 @@ generate_simulated_data <- function(r_in_weeks = # nolint
 
   forecast_date <- date_df |>
     dplyr::filter(.data$t == !!ot + !!nt) |>
-    dplyr::pull("date")
+    dplyr::pull(.data$date)
```

Comment:
Fine as it was. `pull` is tidy selection https://dplyr.tidyverse.org/articles/programming.html
```suggestion
    dplyr::pull("date")
```